meson.build

```diff
@@ -8,20 +8,53 @@ python = import('python')
 # plugin_crypto is enabled if any of the crypto plugins is enabled, auto if
 # none of them are explicitly enabled but at least one is set to auto, or
 # disabled if all of them are disabled.
-plugin_crypto = get_option('plugin-ice')
-foreach plugin : ['plugin-ice', 'plugin-omemo', 'plugin-rtp']
-    if get_option(plugin).enabled() and not plugin_crypto.enabled()
-        plugin_crypto = get_option(plugin)
-    elif get_option(plugin).allowed() and not plugin_crypto.allowed()
-        plugin_crypto = get_option(plugin)
+#
+# On Windows, it's always required because we need it for glib-networking.
+if host_machine.system() == 'windows'
+    plugin_crypto = true
```

Comment:
It might be worth adding this line `add_project_arguments('-D', '_WIN32', language: 'vala')` here, since CMake has such a definition.  After this, the segmentation fault disappeared.

Reply:
Thanks, this one seems to be required. Fixed.